src/vector/mobile_guide/index.css

```diff
@@ -0,0 +1,188 @@
+@import url("@vector-im/compound-design-tokens/assets/web/css/compound-design-tokens.css");
+
+/* By default, hide the custom IS stuff - enabled in JS */
+#custom_is,
+#is_url {
+    display: none;
+}
+
+html {
+    min-height: 100%;
+    position: relative;
+}
+
+body {
+    background: #fff;
```

Comment:
I haven't switched to using `var(--cpd-color-*)` yet as I'd like to figure out the best way to make this page easy to copy over to mobile.element.io.